For util-linux 70b989c: libuuid: rename uuid__generate_* to __uuid_generate_*.

diff -ur e2fsprogs-1.42.1-orig/misc/uuidd.c e2fsprogs-1.42.1/misc/uuidd.c
--- e2fsprogs-1.42.1-orig/misc/uuidd.c	2011-10-09 23:44:45.000000000 +0200
+++ e2fsprogs-1.42.1/misc/uuidd.c	2012-02-25 15:48:35.000000000 +0100
@@ -374,7 +374,7 @@
 			break;
 		case UUIDD_OP_TIME_UUID:
 			num = 1;
-			uuid__generate_time(uu, &num);
+			__uuid_generate_time(uu, &num);
 			if (debug) {
 				uuid_unparse(uu, str);
 				printf(_("Generated time UUID: %s\n"), str);
@@ -384,7 +384,7 @@
 			break;
 		case UUIDD_OP_RANDOM_UUID:
 			num = 1;
-			uuid__generate_random(uu, &num);
+			__uuid_generate_random(uu, &num);
 			if (debug) {
 				uuid_unparse(uu, str);
 				printf(_("Generated random UUID: %s\n"), str);
@@ -393,7 +393,7 @@
 			reply_len = sizeof(uu);
 			break;
 		case UUIDD_OP_BULK_TIME_UUID:
-			uuid__generate_time(uu, &num);
+			__uuid_generate_time(uu, &num);
 			if (debug) {
 				uuid_unparse(uu, str);
 				printf(P_("Generated time UUID %s and "
@@ -414,7 +414,7 @@
 				num = 1000;
 			if (num*16 > (int) (sizeof(reply_buf)-sizeof(num)))
 				num = (sizeof(reply_buf)-sizeof(num)) / 16;
-			uuid__generate_random((unsigned char *) reply_buf +
+			__uuid_generate_random((unsigned char *) reply_buf +
 					      sizeof(num), &num);
 			if (debug) {
 				printf(_("Generated %d UUID's:\n"), num);
